lib/bundle/brewfile.rb

```diff
@@ -2,22 +2,22 @@
 
 module Bundle
   module Brewfile
+    ENV_KEY = "HOMEBREW_BUNDLE_FILE"
     module_function
 
     def path(dash_writes_to_stdout: false)
+      env_bundle_file = ENV.fetch(ENV_KEY, "")
+
       if ARGV.include?("--global")
+        raise "'#{ENV_KEY}' can not be specified with '--global'" unless env_bundle_file.empty?
         Pathname.new("#{ENV["HOME"]}/.Brewfile")
-      else
-        filename = ARGV.value("file")
-        if filename == "-"
-          filename = if dash_writes_to_stdout
-            "/dev/stdout"
-          else
-            "/dev/stdin"
-          end
-        end
-        filename ||= "Brewfile"
+      elsif ARGV.include?("--file")
+        filename = handle_file_value(ARGV.value("file"), dash_writes_to_stdout)
```

Comment:
Looks like you could move the call to `ARGV.value("file")` here into the `handle_file_value` method since that method isn't being invoked elsewhere.

Reply:
That makes sense. I chose to keep `ARGV` access within `#path` so that all access to `ARGV` could be seen at a glance in a single method.

Reply:
I'm fine with either approach.